src/util/mod.rs

```diff
@@ -0,0 +1,14 @@
+//! Self-contained, re-usable utilities that are not specific to this game.
+
+// Unused utilities and re-exports may trigger these lints undesirably.
+// Consider uncommenting these lines.
+// #![allow(dead_code)]
+// #![allow(unused_imports)]
```

Comment:
```suggestion
#![allow(dead_code, unused_imports)]
```
Would be a better default dev experience imo

Reply:
Sure.